website/docs/docs/use-dbt-semantic-layer/exports.md

```diff
@@ -0,0 +1,133 @@
+
+## Define Exports
+
+Exports are an additional configuration added to a Saved Query. They define how to materialize a Saved Query, along with the schema and table name. 
+
+You can define Exports in a YAML format as a key within the `saved_queries` configuration. Here's an example of a Saved Query with an Export:
+
+```yaml
+saved_queries:
+  - name: your_query
+    description: your_description
+    query_params:
+      metrics:
+        - your_metric_name
+      group_bys:
+        - TimeDimension()
+        - ... # Additional group_bys
+      where:
+        - ... # Additional where clauses
+    export:
+      - name: your_export
+        config:
+          export_as: table # Options available: table, view
+          schema: your_schema # Optional - defaults to deployment schema
+          alias: some_table_name # Optional - defaults to Export name
+```
+
+You can use the following parameters to define an Export:
+
+| Parameters | Type    | Required | Description    |
+| ------- | --------- | ---------- | ---------------- |
+| `name` | String    | Required     | Name of the Export object.    |
+| `saved-query` | String    | Required     | A name of a Saved Query that could be used.    |
+| `select` | List or String   | Optional    | Names of Exports to be selected from the Saved Query. |
+| `exclude` | String  | Optional    | Not the names of Exports to be selected from the Saved Query. |
+| `export_as` | String  | Optional    | Type of Export to create from the Export_as types available in the config. |
+| `schema` | String  | Optional    | Schema to use for creating the table or view. |
+| `alias` | String  | Optional    | Table alias to use to write the table or view. |
+
+## Run Exports
+
+Once you've defined Export in your dbt project, the next step is to run them. There are two ways to run an Export:
+  
+1. Run an Export in your development environment using the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation).
+2. Run an Export in your production environment using the [dbt Cloud Job scheduler](/docs/deploy/job-scheduler).
+
+### Exports in Development
+
+You can run an Export in your development environment if you want to test the output of the Export. 
+
+You can run an Export using the dbt Cloud CLI command `dbt sl export` &mdash; for example, `dbt sl export --saved-query sq_name`. This command runs any Export defined for the Saved Query and materialize the table or view in your development environment.
+
+---- ADD OUTPUT OF THE COMMAND HERE ----
+
+By default all Exports are run for a Saved Query. You can use the `select` flag to select a specific Export or you can specify a new Export using the `export-as` flag. In production, the dbt Cloud Job scheduler runs the equivalent of `dbt sl export --saved-query sq_name`.
+
+Since the `--select` flag selects an existing Export, it can't be used with `alias` or `schema` to override the Export configurations. 
+
```

Comment:
what do we mean by 'specify a new export'? do we mean specify a new export name?

Reply:
This would be an entirely new export for this saved query with its own name and confirmation. I won't get written back to the configs, it's just a way to test out what the export would look like in development.